src/app/connector-editor/extension/connectors-filter.data-adapter.ts

```diff
@@ -31,7 +32,9 @@ export class ConnectorsFilterDataAdapter implements FilterDataAdaper {
     }
 
     get contents(): Observable<Connector[]> {
-        return this.store.select(selectProjectConnectorsArray);
+        return this.store.select(selectProjectConnectorsArray).pipe(
+            map(connectors => connectors.filter(connector => !connector.template || connector.template === 'null'))
```

Comment:
are you sure, that this will be the string 'null' and not really a Javascript `null`? Can you please align it with Constantin?

Reply:
@popovicsandras , If it is a js null, it will be filtered by `!connector.template`